yamlpatch/patch_test.go

```diff
@@ -82,6 +82,26 @@ foo:
 foo:
   arr: [1, 2, 3, 4]
   bar: 1 # my bar`,
+		},
+		{
+			Name:  "add to array with comment",
```

Comment:
can we also add a test for an object/map and the test operation?

Reply:
Added a test for `replace` with an object

I looked more at the test operation and realized that the comment isn't relevant / never gets consumed there, so I've stopped passing the comment to that operation entirely.